automated-comments/go/two-fer/replace_stub_comments.md

```diff
@@ -0,0 +1,5 @@
+The stub comments should be replaced with proper comments on package and function.
```

Comment:
How about "The stub comments are placeholders to show where the real documentation comments should go. Replace these with a brief, but helpful, description of the package, and a sentence or two explaining what the function does, what parameters it takes, and what it returns."

Reply:
Make it an action! I took @bitfield 's advice and applied "actionifier"

```suggestion
Replace the stub comments with a brief, but helpful, description of the package, and a sentence or two explaining what the function does, what parameters it takes, and what it returns. `
```

I don't think `stub` needs to be explained.

Reply:
You'd be surprised what does and doesn't need to be explained! I come across this all the time when writing technical books. One of the key things I do is try out drafts on readers of various experience levels, and ask them to point out things that didn't make sense to them. That always results in changes I'd never have guessed needed to be made.

Reply:
In that case leave it in 😅. We're probably targetting experienced developers (in a different language), but even then, you're probably right 😁